src/instructlab/model/serve.py

```diff
@@ -191,3 +194,13 @@ def serve(
     finally:
         backend_instance.shutdown()
         raise click.exceptions.Exit(0)
+
+
+def warn_for_unsuported_backend_param(
```

Comment:
non-blocking nit: would be nice if the function name, or at least docstring made it clear this was for backend == vllm